ILSpy.AddIn/ILSpyAddInPackage.cs

```diff
@@ -119,82 +120,70 @@ private void OpenProjectOutputInILSpyCallback(object sender, EventArgs e)
 
 			foreach (EnvDTE.UIHierarchyItem item in items) {
 				EnvDTE.Project project = (EnvDTE.Project)item.Object;
-				EnvDTE.Configuration config = project.ConfigurationManager.ActiveConfiguration;
-				string projectPath = Path.GetDirectoryName(project.FileName);
-				string outputPath = config.Properties.Item("OutputPath").Value.ToString();
-				string assemblyFileName = project.Properties.Item("OutputFileName").Value.ToString();
-				OpenAssemblyInILSpy(Path.Combine(projectPath, outputPath, assemblyFileName));
+				OpenProjectInILSpy(project);
+			}
+		}
+
+		// Called when the menu is popped, determines whether "Open code in ILSpy" option is available.
+		private void OpenCodeItemInILSpyCallback_BeforeQueryStatus(object sender, EventArgs e)
```

Comment:
This will disable the "Open code in ILSpy" context menu item if, for example, you are editing a text file that is not part of any project.